js/embark.js

```diff
@@ -300,6 +302,38 @@ EmbarkJS.Storage.isAvailable = function () {
   return this.currentStorage.isAvailable();
 };
 
+EmbarkJS.Storage.setProviders = async function (dappConnOptions) {
+  try {
+    await detectSeries(dappConnOptions, async (dappConn, callback) => {
```

Comment:
(note: this code is moved existing code - not new code, from the storage component, into embark.js)

This dapp connections here is the storage equivalent of the one used in contracts.js it will connect to the first one available